cli/package.json

```diff
@@ -50,12 +50,24 @@
     "sort-package-json": "^2.0.0"
   },
   "devDependencies": {
+    "@prisma/client": "^4.8.0",
+    "@tanstack/react-query": "^4.20.4",
+    "@trpc/client": "^10.7.0",
+    "@trpc/next": "^10.7.0",
+    "@trpc/react-query": "^10.7.0",
+    "@trpc/server": "^10.7.0",
     "@types/fs-extra": "^9.0.13",
     "@types/gradient-string": "^1.1.2",
     "@types/inquirer": "^9.0.2",
     "@types/node": "^18.8.0",
+    "next": "^13.1.1",
+    "next-auth": "^4.18.7",
     "prettier": "^2.8.0",
     "prettier-plugin-tailwindcss": "^0.2.0",
+    "prisma": "^4.8.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "superjson": "^1.12.1",
```

Comment:
added all these as devDeps to remove some red squiggles in the template and should also help with autocompletion / prevent typos